src/ConfigProvider.php

```diff
@@ -54,6 +62,45 @@ public function __invoke(): array
                  */
                 'base_path' => null,
             ],
+            'view_manager'       => [
+                /**
+                 * Templates configured here will be provided to the TemplateMapResolver.
+                 * This is conventional for an MVC app
+                 */
+                'template_map' => [
+                    // 'template-name' => 'path/to/template.phtml',
+                ],
+                /**
+                 * Templates configured here will be provided to the TemplatePathStack resolver.
+                 * This is conventional for an MVC app
+                 */
+                'template_path_stack' => [
+                    // 'path/to/a/directory/of/templates',
+                ],
+                /**
+                 * Templates configured here will be provided to the PrefixPathStackResolver
+                 */
+                'prefix_template_path_stack' => [
+                    // 'prefix' => 'path/to/a/directory/of/templates',
+                ],
+                /**
+                 * The default template suffix is configured here in MVC applications
+                 */
+                //'default_template_suffix' => 'phtml',
```

Comment:
Why is this commented out? We can provide our default :thinking:

Reply:
Because this is potentially configured in `view_manager.default_template_suffix` and `templates.extension`, specifying the default in `templates.extension` will make the former impossible to configure which will be confusing for users.

I thought it better to not _configure_ it at all and rely on a fallback constant for when it's completely undefined